.github/workflows/main.yml

```diff
@@ -0,0 +1,20 @@
+name: Bump version
+on:
+  push:
+    branches:
+      - master
+      - develop
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        fetch-depth: '10'
+    - name: Bump version and push tag
+      uses: anothrNick/github-tag-action@1.33.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        WITH_V: true
+        RELEASE_BRANCHES: master
+        TAG_CONTEXT: repo
```

Comment:
Maybe a comment would be good here. What does it do exactly ? Does this generate also the version.py, commit it, push it and tag it ?